inc/ThirdParty/Hostings/Wpengine.php

```diff
@@ -0,0 +1,161 @@
+<?php
+
+namespace WP_Rocket\ThirdParty\Hostings;
+
+use WP_Rocket\ThirdParty\ReturnTypesTrait;
+use WP_Rocket\Event_Management\Subscriber_Interface;
+use WpeCommon;
+
+/**
+ * Compatibility class for WP Engine.
+ *
+ * @since 3.6.1
+ */
+class Wpengine implements Subscriber_Interface {
```

Comment:
What do we think of this class name? Should it be `WPEngine` instead?

I think we need a convention on abbreviation in classe names, like for WP, CSS, JS, etc, to maintain consistency in the codebase

Reply:
Yes, I agree we should keep it uniform and preserve `WP` and the original plugin name `WPEngine`. Modified all files and everything into `WPEngine`